src/components/modulesList/Item.js

```diff
@@ -10,12 +10,7 @@ const ModuleItem = ({ module, onPress }) => {
       <TouchableOpacity style={styles.button} onPress={onPress}>
         <Text style={styles.text}>{module.name}</Text>
       </TouchableOpacity>
-      <Checkbox
-        onPress={() => {
-          console.log("Disabled");
-        }}
-        disabled
-      />
+      <Checkbox disabled />
```

Comment:
Rozumiem że to disabled jest dlatego że jeszcze nie ma implementacji?

Reply:
Tak dokładnie. Nie chcę, żeby użytkownik to klikał, bo ten stan ma zmienić aplikacja, kiedy użytkownik przejdzie przez jakiś moduł